scripts/test-operator-integration.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+# Install multi-operator-manager. This will make sure the latest binary is installed
+# If the installation failed, keep going, maybe the binary is available in the system
+echo "Installing latest version of multi-operator-manager..."
+if ! go install github.com/openshift/multi-operator-manager/cmd/multi-operator-manager@latest; then
+    echo "Error: Failed to install multi-operator-manager."
```

Comment:
should we `exit 1` ?

Reply:
I wanted to keep going because the binary might be available from a previous installation. This is useful for local testing and doesn't required an internet connection to run the target